templates/site.css

```diff
@@ -75,7 +75,7 @@ div.example table {
     margin-top: 15px;
     margin-bottom: 20px;
 }
-p.next {
+p.next p.prev {
```

Comment:
I don't think this CSS does what you expected 😅 Let's revert this change per the comment above about previous links.

Reply:
Ok 😅